### ISSUE TITLE
Command failed with TooFewItemsException: !scry o:'prowess'

### ISSUE THREAD

**Issue by vorpal-buildbot** [PennyDreadfulMTG/Penny-Dreadful-Tools#1459]
```
Traceback (most recent call last):
  File "/home/discord/Penny-Dreadful-Discord-Bot/magic/oracle.py", line 37, in valid_name
    return cards[0].name
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/discord/Penny-Dreadful-Discord-Bot/magic/oracle.py", line 168, in scryfall_import
    valid_name(sfcard['name'])
  File "/home/discord/Penny-Dreadful-Discord-Bot/magic/oracle.py", line 39, in valid_name
    raise InvalidDataException('Did not find any cards looking for `{name}`'.format(name=name))
shared.pd_exception.InvalidDataException: Did not find any cards looking for `Daring Sleuth // Bearer of Overwhelming Truths`

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/discord/Penny-Dreadful-Discord-Bot/discordbot/command.py", line 48, in handle_command
    await method(Commands, bot, message.channel, args, message.author)
  File "/home/discord/Penny-Dreadful-Discord-Bot/discordbot/command.py", line 193, in scryfall
    cards = [cbn.get(name, import_sf(name)) for name in cardnames]
  File "/home/discord/Penny-Dreadful-Discord-Bot/discordbot/command.py", line 193, in <listcomp>
    cards = [cbn.get(name, import_sf(name)) for name in cardnames]
  File "/home/discord/Penny-Dreadful-Discord-Bot/discordbot/command.py", line 191, in import_sf
    if oracle.scryfall_import(name):
  File "/home/discord/Penny-Dreadful-Discord-Bot/magic/oracle.py", line 171, in scryfall_import
    insert_scryfall_card(sfcard)
  File "/home/discord/Penny-Dreadful-Discord-Bot/magic/oracle.py", line 203, in insert_scryfall_card
    CARDS_BY_NAME[name] = load_card(name)
  File "/home/discord/Penny-Dreadful-Discord-Bot/magic/oracle.py", line 42, in load_card
    return load_cards([name])[0]
  File "/home/discord/Penny-Dreadful-Discord-Bot/magic/oracle.py", line 58, in load_cards
    raise TooFewItemsException('Expected `{namelen}` and got `{rslen}` with `{names}`.  missing=`{missing}`'.format(namelen=len(names), rslen=len(rs), names=names, missing=missing))
shared.pd_exception.TooFewItemsException: Expected `1` and got `0` with `{'Bearer of Overwhelming Truths'}`.  missing=`{'Bearer of Overwhelming Truths'}`

```

Reported on Discord by TrotterTime#7689

**Comment by bakert** [comment]
https://github.com/PennyDreadfulMTG/Penny-Dreadful-Tools/commit/44358160547d3362123b3d1b8052371010d46ed5 removed our issue with KeyError but gave us this new one.

List of names that have this problem so far are:

Bearer of Overwhelming Truths
Voracious Reader
Werewolf Ransacker
Ancient of the Equinox
Liliana, Defiant Necromancer
Conqueror's Foothold
Conduit of Emrakul
Ghastly Haunting

aka the reverse side of some transform cards.

**Comment by bakert** [comment]
This bug is not always reproducible which seems to be something to do with the fact that the scryfall command actually writes additional copies of cards to the database in the style of !spoiler which I don't really understand. Before failing, I guess.

Spoiler is ok writing to the db because when new cards are added to mtgjson they will do a version bump and we'll wipe the db. This isn't the case with scryfall search and we end up with (say) two copies of Elusive Tormentor in the face table. This seems bad. We'd have a UNIQUE constraint on face.name if it weren't for the two back halves of meld cards having the same name (kinda).

**Comment by bakert** [comment]
This seems to be ok now. I'm sure mtgbot will let us know if I'm wrong about that.

**Comment by bakert** [comment]
"Fix" was in 405c0207.
